statistics-service/main.go

```diff
@@ -43,5 +44,6 @@ func main() {
 
 	router.Static("/swagger-ui", "./swagger-ui")
 
+	go keptnapi.RunHealthEndpoint("10998")
```

Comment:
Can we also use shipyard-controllers logic for creating a health endpoint with gin gonic?

Reply:
sure, good idea